docs/spark-standalone.md

```diff
@@ -322,9 +322,9 @@ SPARK_WORKER_OPTS supports the following system properties:
   <td>true</td>
   <td>
     Store External Shuffle service state on local disk so that when the external shuffle service is restarted, it will
-    automatically reload info on current executors.  This only affects standalone mode (yarn always has this behavior
-    enabled).  You should also enable <code>spark.worker.cleanup.enabled</code>, to ensure that the state
-    eventually gets cleaned up.  This config may be removed in the future.
+    automatically reload info on current executors.  This only affects standalone mode.  You should also enable 
+    <code>spark.worker.cleanup.enabled</code>, to ensure that the state eventually gets cleaned up.  
+    This config may be removed in the future.
```

Comment:
Why are we removing the yarn related blurb from here ?

Reply:
Hmm... does `yarn always has this behavior enabled` mean that YarnShuffleService will always will persist data into Level/RocksDB?

Isn't that right?

Reply:
- when `yarn.nodemanager.recovery.enabled` is true, `_recoveryPath` and `registeredExecutorFile` in `YarnShuffleService` will not null, then `YarnShuffleService` persist data into Level/RocksDB 
- when `yarn.nodemanager.recovery.enabled` is false, `_recoveryPath` and `registeredExecutorFile` in `YarnShuffleService` will null, then `YarnShuffleService` not persist data into diskstore

YarnShuffleService'` behavior is controlled by Yarn's configuration. It seems that it not related to `spark.shuffle.service.db.enabled`, so I don't think it is necessary to mention `yarn always has this behavior enabled` in this configuration description.

If we need add yarn related descriptions here, should we also need mention `mesos always has this behavior disabled` here...

Reply:
Or can we change 

```
yarn always has this behavior enabled
```

to 

```
The behavior of yarn (and mesos) not related to this configuration
```
?

Reply:
friendly ping @weixiuli @squito to help check this change

Reply:
Have an offline discussion with the author @weixiuli  of this configuration description, [b92cf5c](https://github.com/apache/spark/pull/37853/commits/b92cf5c1e2e4a17a2bad052ed6b0f40cb0dbcb2d) revert change in this pr